src/attribution/onINP.ts

```diff
@@ -37,8 +37,15 @@ const attributeINP = (metric: INPMetric): void => {
       );
     })[0];
 
+    // Currently Chrome can return a null target for certain event types
+    // (especially pointer events). As the event target should be the same
+    // for all events in the same interaction, we pick the first non-null one.
+    // TODO: remove when 1367329 is resolved
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=1367329
+    const firstTargetEntry = metric.entries.find((entry) => entry.target);
+
     (metric as INPMetricWithAttribution).attribution = {
-      eventTarget: getSelector(longestEntry.target),
+      eventTarget: getSelector(firstTargetEntry?.target),
```

Comment:
```suggestion
      eventTarget: getSelector(firstEntryWithTarget && firstEntryWithTarget.target),
```

Reply:
Note that we don't currently use optional chaining anywhere in this repo, and I'm not sure we should introduce it outside of a major version bump (since it could potentially introduce errors for some users where they weren't seeing errors before).